### ISSUE TITLE
basic use of the cluster

### ISSUE THREAD

**Issue by cjayb** [meeg-cfin/sandbox#3]
How do I:
- submit an arbitrary command-line (bash) command to be processed on the cluster
  - `maxfilter`
  - `freesurfer`
  - others?
- find out what is going on with my job
  - is it running/finished
  - why is it taking so long (is the queue full/busy)?
- avoid being yelled at by the admins because I have
  - "submitted a threaded/MPI/MKL/multicore process" (_and what does that even mean?!?_)
  - submitted to the wrong queue (_how many queues are there anyway, and why?!?_)

**Comment by cjayb** [comment]
See also work-in-progress regarding submitting via command-line https://github.com/meeg-cfin/configurations/issues/7

**Comment by MadsJensen** [comment]
Should also be mentioned how to set up a bashrc.

**Comment by cjayb** [comment]
Is this too well hidden? 
http://wiki.pet.auh.dk/wiki/Scientific_computing_basics#Configuration_and_customisation

**Comment by MadsJensen** [comment]
No, but there is very little info on e.g. commands you wrote for isis (e.g. set_mindlabproj) etc.

**Comment by cjayb** [comment]
http://wiki.pet.auh.dk/wiki/Utility_functions_for_CFIN_servers
:)
Perhaps my "conversational" style on the landing page (http://wiki.pet.auh.dk/wiki/Category:MEG) means no-one will find these...

**Comment by MadsJensen** [comment]
ups, I had not seen that :/

**Comment by MadsJensen** [comment]
It should link to [How to use the Grid Engine Linux cluster](http://wiki.pet.auh.dk/wiki/How_to_use_the_Grid_Engine_Linux_cluster).

**Comment by sarathykousik** [comment]
To keep persisting on my own view, we must probably collapse both to a single page?

**Comment by cjayb** [comment]
Persist all you like :) You mean you'd like to see "Scientific computing" and "Utility functions" on the same page? I'm OK with that, rather too many pages than too few! And good landing/redirection/summary pages, not to mention with descriptive names that make them searchable (looking at you, MediaWiki...)

**Comment by sarathykousik** [comment]
Hmm...yeah...  topics are searchable as well..
